src/components/join-flow/email-step.jsx

```diff
@@ -130,7 +131,6 @@ class EmailStep extends React.Component {
                     } = props;
                     return (
                         <JoinFlowStep
-                            description={this.props.intl.formatMessage({id: 'registration.emailStepDescription'})}
```

Comment:
There's a test that expects this to exist. I think that's why travis is failing.

Reply:
Thanks, fixed!